docs/ReleaseNotes.md

```diff
@@ -0,0 +1,57 @@
+# 0.8 Release Notes
+Thanks to all the testers and contributors that helped make version 0.8! We are very excited to have improved many bugs and added new features in version 0.8. 
+
+## Known Issues
+* Wayland support is experimental. In generic packages (Snap, Flatpak, AppImage) there may be extra issues with Wayland. 
+* In generic packages(Snap, Flatpak, AppImage) due to confinement "Open With External Application" does not work. 
+* If close after capture is enabled, and a user copies the image to clipboard, the image may not actually be in the clipboard. We recommend using the "Save" feature with close after capture. 
```

Comment:
I suggest putting `close after capture` in quotation marks. -> "close after capture"